packages/dx-react-chart/src/plugins/zoom-pan.tsx

```diff
@@ -18,9 +18,55 @@ import {
   ScalesCache,
 } from '@devexpress/dx-chart-core';
 import {
-  ZoomAndPanProps, ZoomAndPanState, NumberArray,
+  ZoomAndPanProps, ZoomAndPanState, NumberArray, ZoomPanProviderProps,
 } from '../types';
 
+const events = {
+  wheel: 'onWheel',
+  mousedown: 'onDown',
+  touchstart: 'onDown',
+  touchmove: 'onTouchMove',
+  touchend: 'onTouchEnd',
+};
+
+class ZoomPanProvider extends React.PureComponent<ZoomPanProviderProps> {
+  ref: Element | undefined = undefined;
```

Comment:
It should be declared as

```js
ref!: Element;
```

as we are sure that it will be available when required.